src/main/java/seedu/address/logic/parser/appointmentparser/ViewAppointmentCommandParser.java

```diff
@@ -23,11 +22,11 @@
      */
     public ViewAppointmentCommand parse(String args) throws ParseException {
         try {
-            AppointmentDateTime appointmentDateTime = ParserUtil.parseDateTime(args);
+            AppointmentDateTime appointmentDateTime = ParserUtil.parseDateTime(args + " 00:00");
```

Comment:
Sorry small change needed here, I made the AppointmentDateTime parsing follow more closely with the user guide so an AM/PM is required now. Setting to 12:00AM instead of 00:00 should work.